includes/sdk-browser-supported-version.md

```diff
@@ -0,0 +1,2 @@
+!!! warning "Supported browser versions"
+    This SDK uses [nullish coalescing](https://caniuse.com/mdn-javascript_operators_nullish_coalescing) (`??`) and [optional chaining](https://caniuse.com/mdn-javascript_operators_optional_chaining) (`?.`) operators. Please refer to the links to see supported browser versions.
```

Comment:
Should we make this more general?
`This SDK utilizes modern JavaScript features. Please refer to the provided links for browser compatibility information.
  - nullish coalescing
  - optional chaining
`

Reply:
I like this.